src/components/Nav/Nav.js

```diff
@@ -20,6 +20,13 @@ const Nav = () => {
           </NavLink>
         </IconContext.Provider>
       </Link>
+      <Link to="/results">
+        <IconContext.Provider value={{ color: 'white', size: '7vh' }}>
+          <NavLink>
+            <MdHistory aria-label="ResultsList" />
+          </NavLink>
+        </IconContext.Provider>
+      </Link>
```

Comment:
Your HTML semantics are a bit all over the place here 🙂 There is an ongoing discussion whether it is good to have a list in our nav or not. For only a couple of links it is currently recommended to not use a list and the HTML `nav` element with direct `anchors` as children. You also seem to have a lot of repetition here. How about something like this:
```
function NavBar() {
  const navigationData = [
     {
        path: "/results",
        icon: <MdHistory aria-label={aria}/> /* I think this is possible if I am not mistaken! */
     }
     .... your other data ....
   ]

return (
   <nav>
   {navigationData.map(({path, aria, icon}) => (
         <Link to={path}>
           <IconContext.Provider value={{ color: 'white', size: '7vh' }}>
            {icon}
           </IconContext.Provider>
        </Link>
      )}
   </nav>
```

Reply:
I will try to implement it with the next userstory. So far it didn't seem too repetitive, but as the NavBar grows, your solution seems more practical in the long run.